readme.md

```diff
@@ -51,38 +51,19 @@ Default: `false`
 
 Crop to the set height.
 
-##### delay
-
-Type: `number` *(seconds)*<br>
-Default: `0`
-
-Delay capturing the screenshot. Useful when the site does things after load that you want to capture.
-
 ##### timeout
 
 Type: `number` *(seconds)*<br>
 Default: `60`
 
-Number of seconds after which PhantomJS aborts the request.
+Number of seconds after which Google Chrome aborts the request.
 
 ##### selector
 
 Type: `string`
 
 Capture a specific DOM element.
 
-##### css
```

Comment:
Weird that we don't have a replacement for this... Should we open a new issue in Puppeteer?

Reply:
Yes, open an issue on Puppeteer. In the meantime, we can use `page.addScriptTag(url)` to inject the CSS.

Reply:
We can probably do this in [`page.evaluate()`](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pageevaluatepagefunction-args) too.

Reply:
https://github.com/GoogleChrome/puppeteer/issues/892

Reply:
PR sent https://github.com/GoogleChrome/puppeteer/pull/947